s4e-web/cypress/integration/map/map-data-search.spec.ts

```diff
@@ -0,0 +1,55 @@
+/// <reference types="Cypress" />
+
+import { Login } from '../../page-objects/auth/auth-login.po';
+import { MapDataSearch } from '../../page-objects/map/map-data-search.po';
+
+before(function () {
+  cy.fixture('users/zkMember.json').as('zkMember');
+});
+
+describe.only('Map Data Search', () => {
+
+  context("Data search for not logged in user", () => {
+
+    before(() => {
+      cy.visit('/login');
+    });
+
+    it('shouldn\'t download data without authentication', () => {
+      Login
+        .goToMapWithoutLogin()
+      MapDataSearch
+        .goToSearchData()
+        .selectNthProductToSearch(0)
+        .searchData()
+        .selectNthDataToDownload(0)
+        .shouldReturnToLoginPage()
```

Comment:
Wouldn't be better to replace this function with simple cy.location('path').should('be', '/login')?

Reply:
every assertion are in PO, so I use method for that